vita3k/util/include/util/log.h

```diff
@@ -69,5 +71,3 @@ std::string log_hex(T val) {
     using unsigned_type = typename std::make_unsigned<T>::type;
     return fmt::format("0x{:0X}", static_cast<unsigned_type>(val));
 }
-
-void dump_hex(const std::vector<uint8_t> &bytes, std::ostream &stream);
```

Comment:
I speek here end file finish by } and no have new line

Reply:
Yes I know and it has one. A missing line at the end of a file has the sign ![image](https://user-images.githubusercontent.com/25406440/107421293-158fcf00-6b1a-11eb-8a54-d7b5049240f9.png) or do you want one more empty line?

Reply:
All end file request one new line after code

Reply:
Yes and it has one. I can't add an other line, clang-format doesn't want it.

Reply:
Oh strange